### DISCUSSION TITLE
question about index-hopping in γδ TCR sequences

### DISCUSSION THREAD

**Discussion by haoqing12** [milaboratory/mixcr#1061] [Q&A]
Hello,
Thanks for developing a state-of-the-art tool.

I have a few questions in practice:

1, I have γδTCR sequencing conducted by iGeneTech Multi-IR Library Prep Kit, which is an amplicon protocol based on multiplex primers ( v-primers and j-primers ) without UMI. (PE 2x150 on an Illumina platform) The material is RNA from human PBMC. 

This kit does not include in your guides, so would you mind to inform of what would be the best commands to apply my TCR sequence?

2, PCR error and index-hopping issue in the demultiplexing step are the two key factors that affect the accuracy of γδ TCR sequencing, especially without UMI. I noticed that PCR error was considered in the tool.  So, I am curious did Mixcr filter out such index-hopping issues in the analysis steps?

Thanks in advance
haoq

**Comment by mizraelson** [comment]
Hi, thank you!

We will add this kit to our presets. Is there a scheme with primers location? Specifically what FR or CDR region the forward primers are located?

As for now you can use a generic preset in the following way:

```
mixcr analyze generic-tcr-amplicon \
    --species hsa \
    --rna \
    --floating-left-alignment-boundary \
    --floating-right-alignment-boundary J \
      input_R1.fastq.gz \
      input_R2.fastq.gz \
      result
```

This command will assemble clones by `CDR3` feature. If the library also covers some more distant parts of the V gene and you want to assemble clones by a longer feature, you can add the following parameter:

`--assemble-clonotypes-by {FR2Begin:CDR3End}`

As for your second question, if the samples have been already de-multiplexed there is not much we can do from there. But with non de-multiplexed data there is a way which should potentially work, but that would be a bit more custom approach and requires a custom dedicated preset. Let me know if you  need help with it.

**Reply by haoqing12** [reply]
Thank you very much.
My data has been already de-multiplexed. So, maybe I can use more strict criteria: such as only clonotypes with clone size >= 2  can be used in computing the overlap. 

Primer details will be posted here after company reply.
